.circleci/config.yml

```diff
@@ -30,14 +30,13 @@ jobs:
           command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/01-fusion-standardization.R -f "data/pbta-fusion-starfusion.tsv.gz" -c "STARfusion" -o "scratch/star.RDS" 
                                                 
       - run:
-          name: Unsupervised Transcriptomic Analysis - Data Prep
-          command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/01-transcriptomic-analysis-prep.R --perplexity 1 --neighbors 15
-                
+          name: Unsupervised Transcriptomic Analysis - Subset by Strategy          
+          command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/01-subset-expression-by-strategy.R
+    
       - run:
-          name: Unsupervised Transcriptomic Analysis - Plotting
-          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/transcriptomic-dimension-reduction/02-transcriptomic-analysis-plotting.Rmd', clean = TRUE)"
+          name: Unsupervised Transcriptomic Analysis - Run Dimension Reduction         
+          command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/scripts/run-dimension-reduction.R  --expression data/pbta-gene-expression-rsem.fpkm.rds --metadata data/pbta-histologies.tsv --filename_lead rsem_all --output_directory analyses/transcriptomic-dimension-reduction/results --skip_tsne
                 
-
         #### Add your analysis here ####
```

Comment:
Is the idea now to create a shell script that would sequentially execute `01-subset-expression-by-strategy.R`, `02-dimension-reduction.sh`, and `03-transcriptomic-analysis-plotting.Rmd`?

Reply:
Also, `03-transcriptomic-analysis-plotting.Rmd` may need to be renamed to something that agrees better with the other scripts. Perhaps, `03-plot-dimension-reduction.Rmd` may be more appropriate.

Reply:
> Is the idea now to create a shell script that would sequentially execute 01-subset-expression-by-strategy.R, 02-dimension-reduction.sh, and 03-transcriptomic-analysis-plotting.Rmd?

Yes and probably another script specifically for CI where t-SNE is skipped.

> Also, 03-transcriptomic-analysis-plotting.Rmd may need to be renamed to something that agrees better with the other scripts. Perhaps, 03-plot-dimension-reduction.Rmd may be more appropriate.

That will be part of part 2 :)